golang/cosmos/x/vtransfer/ibc_middleware_test.go

```diff
@@ -474,6 +485,42 @@ func (s *IntegrationTestSuite) TestHops() {
 					receiver = baseReceiver
 				}
 
+				// Reset the chain state.
+				var modifiedPacketData []byte
+				mspd := func(ctx sdk.Context, cdc codec.Codec, data []byte) ([]byte, error) {
+					newData, err := s.modifySendPacketData(cdc, data, sender)
```

Comment:
Comment that we need to put hooked sender back in so that vtransfer keeper can strip it back out as if it had been there all along.

Consider changing this to "overwriteSendPacketData" or something like that.

Reply:
Done.